### PR TITLE
update test files

### PR DESCRIPTION
**Pull request by willdurand**
I updated the following files by pulling the most recent version from GitHub:

- `bin/install-wp-tests.sh`
- `tests/bootstrap.php`

I added the polyfill that is now required in `composer.json`, and updated the lockfile.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -23,11 +23,11 @@ $ docker run --name mysql_addons_wp_headless -e MYSQL_ROOT_PASSWORD=pass -e MYSQ
 Assuming you have a MySQL server running locally using the docker command above, run the following commands once:
 
 ```
-# `5.7.1` is the WordPress version, use `latest` for the latest version or any
+# `5.8.3` is the WordPress version, use `latest` for the latest version or any
 # other version if you like.
 # `true` at the end of the command below skips the database creation since it is
 # created when the docker container starts
-$ ./bin/install-wp-tests.sh addons_wp_headless root pass '127.0.0.1:55001' 5.7.1 true
+$ ./bin/install-wp-tests.sh addons_wp_headless root pass '127.0.0.1:55001' 5.8.3 true
```
Comment (eviljeff) [inline README.md:30]: the circleci tests are still running on 5.6.2 though (not sure why 5.7.1 currently), so should the readme match?

**Comment by willdurand** [comment]
> I updated the following files by pulling the most recent version from GitHub:

This wasn't strictly needed. I pushed the updates because I had the updated files locally.

File: composer.json [hunk 2]
```diff
@@ -1,5 +1,6 @@
 {
-    "require": {
-        "phpunit/phpunit": "5.7.*"
+    "require-dev": {
+        "phpunit/phpunit": "^5.7.21 || ^6.5 || ^7.5",
```
Comment (eviljeff) [inline composer.json:3]: what's the logic behind these specific versions?

**Review by eviljeff — APPROVED** [review]
just some versioning related queries

File: composer.json (hunk 2 above)
Comment (willdurand) [reply composer.json:3]: These constraints are the 3 different major versions that are currently supported by WordPress.

File: README.md (hunk 1 above)
Comment (willdurand) [reply README.md:30]: Yeah, I can adjust that.

**Pull request merged**